### PR TITLE
rename greet to checkRepositoryAccess

### PR DESCRIPTION
**Pull request by Echidna93**
# Pull Request Description

This pull request closes #

# Acceptance Test

* [ ] Building the code with `mvn clean install -Dintegration.tests` still works.
* [ ] Running `mvn spring-boot:run` in the `strongbox-web-core` still starts up the application correctly.
* [ ] Building the code and running the `strongbox-distribution` from a `zip` or `tar.gz` still works.
* [ ] The tests in the [`strongbox-web-integration-tests`](https://github.com/strongbox/strongbox-web-integration-tests/) still run properly.

# Questions

* Does this pull request break backward compatibility? 
  * [ ] Yes
  * [ ] No

* Does this pull request require other pull requests to be merged first? 
  * [ ] Yes, please see #...
  * [ ] No

* Does this require an update of the documentation?
  * [ ] Yes, please see strongbox/strongbox-docs#{PR_NUMBER}
  * [ ] No

### CODE REVIEW

File: strongbox-web-core/src/main/java/org/carlspring/strongbox/controllers/BaseArtifactController.java [hunk 1]
```diff
@@ -54,5 +55,9 @@ else if (request.getMethod().equals(RequestMethod.HEAD.name()))
 
         return true;
     }
+    public ResponseEntity<String> checkRepositoryAccess(){
+	
+	    return new ResponseEntity<>("success", HttpStatus.OK);
+    }
```
Comment (carlspring) [inline strongbox-web-core/src/main/java/org/carlspring/strongbox/controllers/BaseArtifactController.java:61]: Please, follow our [coding convention](https://strongbox.github.io/developer-guide/coding-convention.html):
```suggestion
    public ResponseEntity<String> checkRepositoryAccess()
    {
        return new ResponseEntity<>("success", HttpStatus.OK);
    }
```

**Comment by carlspring** [comment]
Hi!

Thank you for your contribution!

Would you mind signing the ICLA, as described in the [Contributing](https://github.com/strongbox/strongbox/blob/master/CONTRIBUTING.md) page?

Also, please, feel free to join our [chat channel](https://chat.carlspring.org/), if you'd like to learn more about the project and/or like to find out what else you could help with.

Kind regards,

Martin

**Comment by Echidna93** [comment]
Hi,

I have a signed ICLA in the mail that should be arriving soon. 

Alex

**Comment by carlspring** [comment]
> I have a signed ICLA in the mail that should be arriving soon.

I assume you're referring to e-mail, right? If so, I have not received it.

**Comment by carlspring** [comment]
@Echidna93 ,

I'm afraid I still haven't received your ICLA.

**Comment by Echidna93** [comment]
Hi @carlspring, 

I sent it via snailmail "If you have not already done so, please complete and send an original signed Agreement to Carlspring Consulting & Development Ltd., 4 Point Close, London, United Kingdom, SE10 8QS." Do you want it via email as well? Sorry for the confusion.

Alex

**Comment by Echidna93** [comment]
@carlspring, I'll get that to you today. Sorry about that.

**Comment by Echidna93** [comment]
@carlspring you should have a signed ICLA in your gmail inbox now.

**Comment by carlspring** [comment]
@Echidna93 ,

Could you please rebase your branch and apply the requested changes? Thanks! :)

**Comment by Echidna93** [comment]
Yes, absolutely. I'll do that today. I'll also start working on refactoring the other controller's greet methods as well.

**Comment by carlspring** [comment]
Closed by #1817 .

**Pull request closed without merging**